### PR TITLE
interview count lambda

### PR DESCRIPTION
**Pull request by snirkap**

### CODE REVIEW

File: backend/lambda_functions/interview_count_lambda.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import json
+import boto3
```
Comment (entrypoint-community) [inline backend/lambda_functions/interview_count_lambda.py]: Please create a requirements file for needed libraries - `requests, boto3`

File: backend/lambda_functions/interview_count_lambda.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import requests
+from datetime import datetime, timedelta
+
+cloudwatch = boto3.client('cloudwatch')
+
+def lambda_handler(event, context):
```
Comment (entrypoint-community) [inline backend/lambda_functions/interview_count_lambda.py]: Please devide the code under the `lambda_handler` function into smaller functions for better readability.

File: backend/lambda_functions/interview_count_lambda.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    # we need to Define the real CloudWatch metric to retrieve
+    metric_name = 'InterviewCount'
+    namespace = 'Interviews'
+    company_name = event.get('company_name')
+    interview_date = event.get('interview_date')
+    
+    if not company_name or not interview_date:
```
Comment (entrypoint-community) [inline backend/lambda_functions/interview_count_lambda.py]: This is good, but we want to be able to pull all existing metrics without specifying the `date` and `company name`.
Please add the ability to pull all existing metrics.

File: backend/lambda_functions/interview_count_lambda.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    
+    payload = {
+        'company_name': company_name,
+        'interview_date': interview_date.strftime('%Y-%m-%d'),
+        'interview_count': interview_count
+    }
+    
+    response = requests.post(website_url, json=payload)
+    
+    if response.status_code == 200:
```
Comment (entrypoint-community) [inline backend/lambda_functions/interview_count_lambda.py]: Our lambda will be triggered from API GW, which is CORS enabled, please add the CORS access origin header.

**Comment by snirkap** [comment]
i added  requirements.txt and fix lambda

**Review by entrypoint-community — APPROVED** [review]

**Pull request merged**